templates/default/choose/chooselocation.html

```diff
@@ -319,9 +322,12 @@
 {/if}
 
 // select city and street from selects
-$( '#state_sel'     ).val( {$data.stateid} );
-$( '#district_sel'  ).val( {$data.districtid} );
-$( '#city_sel'      ).val( {$data.cityid} );
+if (typeof($data) != 'undefined' && $data.length > 0) {
+    $( '#state_sel'     ).val( {$data.stateid} );
+    $( '#district_sel'  ).val( {$data.districtid} );
+    $( '#city_sel'      ).val( {$data.cityid} );
+}
+
```

Comment:
`{$data}` pochodzi ze smarty, a tu używasz jakby to były natywne zmienne JS?

Reply:
https://github.com/chilek/lms/pull/2266/commits/9dd0101cc7a73ed2d815c9135a12138f0ae82932